Source/WebKit/NetworkProcess/storage/CacheStorageStore.h

```diff
@@ -33,7 +33,7 @@ namespace WebKit {
 struct CacheStorageRecord;
 struct CacheStorageRecordInformation;
 
-class CacheStorageStore : public RefCounted<CacheStorageStore> {
+class CacheStorageStore : public ThreadSafeRefCounted<CacheStorageStore> {
```

Comment:
What thread should CacheStorageStore be destroyed on? `CacheStorageDiskStore` is a subclass and has String data members. As a result, it cannot be constructed on thread A and be destroyed on thread B. If CacheStorageStore is always expected to be constructed / destroyed on the main runloop, then we'd want to use `DestructionThread::MainRunLoop` as second template parameter to make sure.

Reply:
It's constructed on WorkQueue.